.github/workflows/destroy.yaml

```diff
@@ -59,7 +59,6 @@ jobs:
           echo object_id=\""$OBJECT_ID"\" >> terraform.tfvars
           echo use_oidc=true >> terraform.tfvars
           echo resource_group_name=\""$RESOURCE_GROUP_NAME"\" >> backend.tfvars
-          echo storage_account_name=\"phditfstate"${CLIENT_ID:0:8}"\" >> backend.tfvars
```

Comment:
Not that we ever use the destroy workflow, but I think we actually still need this line there

Reply:
added back